pages/common/cpio.md

```diff
@@ -0,0 +1,16 @@
+# cpio
+
+> Copies files in and out of archives.
+> Supports following archive formats: binary, old ASCII, new ASCII, crc, HPUX binary, HPUX old ASCII, old tar, and POSIX.1 tar.
+
+- Take a list of file names from standard input and copy them [o]ut to an archive in binary format:
```

Comment:
"copy out to an archive" does sound awkward. "copy out of" would make sense but not "copy out to". Also, it seems strange that a redirection is needed even with the -o option, but hey, if that's how it works...

By the way, "binary format" is somewhat hand-wavy. We could perhaps mention in the main description that it is a *custom* binary format.